pages/contribute/how_to_contribute.md

```diff
@@ -7,120 +7,21 @@ toc: false
-<text transform="matrix(1 0 0 1 225.5213 424.6697)" class="st4 st5">section of our guide</text>
-<text transform="matrix(1 0 0 1 380.6198 600.3485)" class="st4 st5">Get notified when your </text>
-<text transform="matrix(1 0 0 1 368.8198 626.7485)" class="st4 st5">contribution is published!</text>
-<text transform="matrix(1 0 0 1 172.9423 288.1902)" class="st4 st5">Yes</text>
-<text transform="matrix(1 0 0 1 306.3673 186.1899)" class="st4 st5">No</text>
-<text transform="matrix(1 0 0 1 680.3296 55.853)" class="st4 st5">Google docs</text>
-<text transform="matrix(1 0 0 1 169.0636 51.3588)" class="st4 st5">Git + Markdown</text>
-<g>
-	<g>
-		<polyline class="st2" points="344,67.8 148,67.8 148,158.6 		"/>
-		<g>
-			<path d="M148,160.3c-2.1-5.7-5.7-12.7-9.5-17.1l9.5,3.4l9.5-3.4C153.7,147.5,150.1,154.6,148,160.3z"/>
-		</g>
-	</g>
-</g>
-<g>
-	<g>
-		<polyline class="st2" points="640,67.8 836,67.8 836,350.1 		"/>
-		<g>
-			<path d="M836,362.1c-2.1-5.7-5.7-12.7-9.5-17.1l9.5,3.4l9.5-3.4C841.7,349.4,838.1,356.4,836,362.1z"/>
-		</g>
-	</g>
-</g>
-<g>
-	<g>
-		<polyline class="st2" points="147.9,246.7 147.9,311.3 317,311.3 317,350.1 		"/>
-		<g>
-			<path d="M317,362.1c-2.1-5.7-5.7-12.7-9.5-17.1l9.5,3.4l9.5-3.4C322.7,349.4,319.1,356.4,317,362.1z"/>
-		</g>
-	</g>
-</g>
-<polyline class="st2" points="491.9,246.1 491.9,311.3 317,311.3 "/>
-<g>
-	<g>
-		<polyline class="st2" points="317,449.1 317,521.3 491.7,521.3 491.9,552 		"/>
-		<g>
-			<path d="M492,564.1c-2.2-5.7-5.8-12.7-9.7-17l9.5,3.4l9.5-3.5C497.6,551.3,494.1,558.4,492,564.1z"/>
-		</g>
-	</g>
-</g>
-<polyline class="st2" points="836,448.1 836,521.3 486,521.3 "/>
-<text transform="matrix(1 0 0 1 68.0755 18.0215)" class="st6 st4 st5">Our prefered way</text>
-</svg>
-<figcaption>
-Click on the paths in the diagram to go to the different sections.
-</figcaption>
-</figure>
 
-* **Github way:** We use GitHub to develop the RDM toolkit website. GitHub allows us to accept and coordinate contributions under an open authoring and review process. Therefore, this is our preferred way of accepting contributions. To begin with, you will need a personal GitHub account, which you can create on the [GiHub Join page](https://github.com/join).   
-* **Google Doc way:** Alternatively, we provide Google Doc templates. This approach requires you to email our editors, who will transfer your content to our git repository for you.
+Everyone is welcome to contribute to this site, and we've tried to make it as easy as possible. Choose a way that suits you. The website is hosted on GitHub, so:
+
+* If you are not familiar with GitHub but want to give it a try: follow [our step-by-step instructions](github_way) (no technical knowledge required!).
+* If you are happier using Google Docs: follow [the Google Doc way](google_doc_way). 
+* If you are familiar with Github: [fork the repo and create a pull request](working_with_git).
```

Comment:
```suggestion
* If you are familiar with Git: [fork the repo and create a pull request](working_with_git).
```

Reply:
@bedroesb I agree with the use of Git instead of Github. I remember we had Git initially but @nsoranzo had suggested Github.

Reply:
My suggestion was for the "GitHub way" method, which doesn't make any use of Git directly, but in this other sentence is fine.

Reply:
> My suggestion was for the "GitHub way" method, which doesn't make any use of Git directly, but in this other sentence is fine.

thanks @nsoranzo we can merge this then.